src/Relation/Binary/Construction/Free/Pointed/Pointwise.agda

```diff
@@ -0,0 +1,57 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Consequences on pointwise equality of freely adding a point to a Set
+------------------------------------------------------------------------
+
+open import Relation.Binary
+
+module Relation.Binary.Construction.Free.Pointed.Pointwise
+       {a e} {A : Set a} (_≈_ : Rel A e) where
+
+open import Function
+open import Function.Equivalence using (equivalence)
+open import Relation.Nullary
+import Relation.Nullary.Decidable as Dec
+import Relation.Binary.PropositionalEquality as P
+open import Relation.Binary.Construction.Free.Pointed
+
+data _≈∙_ : Rel (Pointed A) e where
```

Comment:
This whole file is really just `Eq` from `Data.Maybe` right?

Hmm `Data.Maybe` needs splitting up into a proper hierarchy as well.

Reply:
TIL. Didn't know about `IsDecEquivalence` either.

Reply:
It's not quite the same thing though: here `Eq` takes its parameter via the module's parameters. Meaning that once you've opened the module with a particular binary relation the constructors are specialized for that relation.

Cf. agda/agda/issues/3210 where I argue we should be able to recover something like `_≈∙_` from `Eq`.